qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider.hpp

```diff
@@ -30,52 +30,53 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef qt_gui_cpp__RosPluginlibPluginProvider_HPP
-#define qt_gui_cpp__RosPluginlibPluginProvider_HPP
+#ifndef QT_GUI_CPP__ROS_PLUGINLIB_PLUGIN_PROVIDER_HPP_
+#define QT_GUI_CPP__ROS_PLUGINLIB_PLUGIN_PROVIDER_HPP_
 
 // Pluginlib has an optional dependency on boost::shared_ptr, which is not required here
 // On machines without boost, including pluginlib/class_loader.hpp requires defining this flag to
 // disable that dependency. Mosty notably these are the machines configured on ci.ros2.org
 #define PLUGINLIB__DISABLE_BOOST_FUNCTIONS
```

Comment:
While we are here, we can remove this `#define`.  This hasn't been necessary since 2020.

```suggestion
```